frontend/src/Editor/QueryManager/QueryEditors/Googlesheets.jsx

```diff
@@ -117,6 +118,24 @@ class Googlesheets extends React.Component {
           </div>
       )}
 
+      {this.state.options.operation === 'delete' && (
```

Comment:
id the of the oepration should be `delete_row` instead of `delete` so that it won't be confusing when we add the 'delete spreadsheet' operation.

Reply:
Thanks @Navaneeth-pk  for pointing this out.